wizeline/remix-project-lab
# Feat - 188 - Resource/Provider Filter for projects screen

# What does this PR do?

Adds filters for Resource and Provider. The provider filter is only available if a Resource filter is selected first. This also means that if the Resource filter is cleared, the provider clear linked to that filter is cleared as well.

This also fixes an annoying warning that appeared on the browser console related to the ProposalCard rendering an `<a>` tag inside another `<a>` tag

# Where should the reviewer start?

app/models/project.server.ts
app/routes/projects/index.tsx

# How should this be manually tested?

Go to the Projects page
Add some resources to a project(s)
Look for the new `Resources` filter on the left side
Click on a resource filter and check that only shows projects with that type of resource.
After selecting a resource a second filter appears called `Provider`
Click a provider filter and check that the results are correct

# Any background context you want to provide?



# What are the relevant tickets?

[Ticket 188](https://github.com/wizeline/remix-project-lab/issues/188)

# Screenshots

![image](https://github.com/wizeline/remix-project-lab/assets/75454591/dba5ac8e-7621-496d-8c82-e5aca3c2f8f2)

![image](https://github.com/wizeline/remix-project-lab/assets/75454591/825a26d5-c9cd-4e42-a960-875a08da01f1)

![image](https://github.com/wizeline/remix-project-lab/assets/75454591/c9f24755-de38-4ec9-8c9a-9381cb1ce547)


# Questions



# Checklist
